toolkit/scripts/toolchain.mk

```diff
@@ -207,15 +207,14 @@ $(toolchain_rpms):
 		wget $$url/$$rpm_filename \
 			$(if $(TLS_CERT),--certificate=$(TLS_CERT)) \
 			$(if $(TLS_KEY),--private-key=$(TLS_KEY)) \
-			-a $$log_file \
-			&& \
-		echo "Downloaded toolchain RPM: $$rpm_filename" >> $$log_file && \
+			-a $$log_file && \
+		echo "Downloaded toolchain RPM: $$rpm_filename" | tee -a $$log_file && \
```

Comment:
> | tee -a  [](http://example.com/codeflow?start=49&length=10)

That was the previous version but it turned out to be more noisy than helpful. I'd suggest reverting that.